src/components/Article/Article.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import IArticle from '../../shared/modules/article';
```

Comment:
Rank C: Không cần prefix `I` cho interface. Hiểu đơn giản đối tượng ở đây là `Article`, ko phải `IArticle`
Có thể tham khảo: https://stackoverflow.com/questions/31876947/confused-about-the-interface-and-class-coding-guidelines-for-typescript